runtime/vm/ContinuationHelpers.cpp

```diff
@@ -215,6 +215,7 @@ copyFieldsFromContinuation(J9VMThread *currentThread, J9VMThread *vmThread, J9VM
 	vmThread->stackObject = continuation->stackObject;
 	vmThread->entryLocalStorage->oldEntryLocalStorage = continuation->oldEntryLocalStorage;
 	vmThread->entryLocalStorage->jitGlobalStorageBase = (UDATA*)&continuation->jitGPRs;
+	vmThread->entryLocalStorage->i2jState = continuation->i2jState;
```

Comment:
`vmThread->entryLocalStorage` is being accessed multiple times. It can be stored in a local to reduce the number of pointer dereferences.

Reply:
fixed

Reply:
I meant the following:
```suggestion
	els->oldEntryLocalStorage = continuation->oldEntryLocalStorage;
	els->jitGlobalStorageBase = (UDATA*)&continuation->jitGPRs;
	els->i2jState = continuation->i2jState;
	vmThread->entryLocalStorage = els;
```
Line 206 has been relocated.